web-ui/src/main/resources/catalog/locales/fr-admin.json

```diff
@@ -180,6 +180,7 @@
     "data.configDir": "Répertoire pour la configuration :",
     "data.dataDir": "Répertoire des données :",
     "data.formatterPath": "Répertoire des mises en page",
+    "data.htmlCacheDir": "Répertoire de HTML cache:",
```

Comment:
Should not be done in Transifex?

Reply:
We will have to translate it in transifex once the PR is merged but having it here is not really a problem. It will be overwritten